### PR TITLE
Randomize the key balanced 

### PR DESCRIPTION
**Pull request by fdurand**
# Description
Randomize the key balanced calculation algorithm.
Let's say you have 1 internal server that proxy to 2 radius server then proxy to 2 others then with the modulo calculation the first proxy servers will always send the request to an only one server.
This code will allow to randomize the KeyBalanced attribute to load-balance (50/50) to the next servers.

# Impacts
Radius Proxy flow

# Delete branch after merge
YES

# Checklist
- [ ] Document the feature
- [ ] Add unit tests
- [ ] Add acceptance tests (TestLink)

# NEWS file entries

## Enhancements
* Randomize KeyBalanced to randomize the load balance in Freeradius Proxy.

### CODE REVIEW

File: conf/pf.conf.defaults [hunk 1]
```diff
@@ -1155,6 +1155,11 @@ normalize_radius_machine_auth_username=disabled
 # Which attributes to use to get the username from a RADIUS request
 # The order of the attributes are listed in this configuration parameter is followed while performing the lookup
 username_attributes=PacketFence-UserNameAttribute,TLS-Client-Cert-Subject-Alt-Name-Upn,TLS-Client-Cert-Common-Name,User-Name
+#
+# radius_configuration.forward_key_balanced
+#
+# Send in the proxy request the attribute PacketFence-KeyBalanced to randomize the load balance in the next PacketFence RADIUS server
+PacketFence-KeyBalanced=disabled
```
Comment (julsemaan) [inline conf/pf.conf.defaults]: That parameter name doesn't follow the convention in pf.conf.defaults

And also, not the same name as in documentation.conf (which is fine there)

**Comment by jrouzierinverse** [comment]
Was this tested?

**Comment by fdurand** [comment]
Not in production

**Comment by jrouzierinverse** [comment]
Let's move to 9.1?

**Comment by extrafu** [comment]
I think we can move it to v9.1 and assess later if we have time to get it into v9.

**Comment by fdurand** [comment]
Bump

File: conf/documentation.conf [hunk 2]
```diff
@@ -1607,6 +1607,13 @@ description=<<EOT
 Treat OCSP response errors as 'soft' failures and still accept the certificate.
 EOT
 
+[radius_configuration.forward_key_balanced]
+type=toggle
+options=enabled|disabled
+description=<<EOT
+Send in the proxy request the attribute PacketFence-KeyBalanced to randomize the load balance in the next PacketFence RADIUS server
```
Comment (julsemaan) [inline conf/documentation.conf]: Can you just add a brief description to explain the benefit of using that attribute like:
"Used to ensure requests are balanced when they come from @lzammit's sandwich"

**Pull request merged**